extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -526,6 +527,79 @@ describes.realWin(
             'info': 'Entering fullscreen.',
           });
         });
+
+        describe('actionPromptTrigger', () => {
+          it('should not expand when actionPromptTrigger is false', async () => {
+            consentUI = new ConsentUI(mockInstance, {
+              'promptUISrc': 'https//promptUISrc',
+            });
+
+            consentUI.show(false);
+            consentUI.iframeReady_.resolve();
+            await macroTask();
+
+            // Send expand
+            sendMessageConsentUi(consentUI, 'enter-fullscreen');
+
+            // not currently fullscreen
+            expect(consentUI.isFullscreen_).to.be.false;
+            expect(consentUI.isActionPromptTrigger_).to.be.false;
+            expect(
+              consentUI.parent_.classList.contains(
+                consentUiClasses.iframeFullscreen
+              )
+            ).to.be.false;
+          });
+
+          it('should expand when actionPromptTrigger is true', async () => {
+            consentUI = new ConsentUI(mockInstance, {
+              'promptUISrc': 'https//promptUISrc',
+            });
+
+            consentUI.show(true);
+            consentUI.iframeReady_.resolve();
+            await macroTask();
+
+            // Send expand
+            sendMessageConsentUi(consentUI, 'enter-fullscreen');
+
+            expect(consentUI.isFullscreen_).to.be.true;
+            expect(consentUI.isActionPromptTrigger_).to.be.true;
+            expect(
+              consentUI.parent_.classList.contains(
+                consentUiClasses.iframeFullscreen
+              )
+            ).to.be.true;
+          });
+
+          it('should expand after hidden', async () => {
```

Comment:
What's this test for? And can we remove it?

Reply:
This is more like an e2e test to test a common workflow (consent prompt is shown on-load, user makes decision, user then clicks button that calls `consentUi#show(true)` consent prompt enters full screen). 

Do you think it should be removed?

Reply:
I would rather have it removed. We are mocking many services in unit tests, a workflow doesn't make too much sense unless there's a corner case scenario or race condition you want to fake. 
My understanding is that everything before the second `consentUI.show(true)` doesn't affect the test at all. which makes it a duplicate test case.

Reply:
SGTM